frontend/components/AdminItemsTable.js

```diff
@@ -55,7 +55,9 @@ const AdminItemsTable = ({ items }) => {
                 <td>{item.description}</td>
                 <td>{item.price}</td>
                 <td>
-                  <Button type="button">Edit</Button>
+                  <a href={`/admin-edit-item?id=${item._id}`}>
+                    <Button type="button">Edit</Button>
+                  </a>
```

Comment:
Check out the next documentation for how to link between pages in a next.js app
https://nextjs.org/docs/api-reference/next/link
I pretty much does an anchor tag under the hood, but it also does nice stuff with server-side rendering for making the app nice and fast between page changes etc.